.github/ISSUE_TEMPLATE/3.feature_request.yaml

```
@@ -0,0 +1,44 @@
+name: Feature request
```

```suggestion
name: Feature Request
```

minor nit since "Usage Question" is title-cased